frontend/appflowy_flutter/lib/workspace/application/settings/shortcuts/settings_shortcuts_cubit.dart

```diff
@@ -40,7 +40,8 @@ class ShortcutsCubit extends Cubit<ShortcutsState> {
         customizeShortcuts,
       );
       //sort the shortcuts
-      commandShortcutEvents.sort((a, b) => a.key.compareTo(b.key));
+      commandShortcutEvents
+          .sort((a, b) => a.key.toLowerCase().compareTo(b.key.toLowerCase()));
```

Comment:
Doesn't this make the shortcuts quite messy in the settings?

Shouldn't we have some kind of grouping mechanism or similar, to make it easier to scroll down and see which settings do what.

Some groups could eg. be: `Selection`, `Quick actions`, `Navigation`, etc.

Reply:
Great suggestion! Actually, we have planned Sorting and Filtering of the shortcuts in the future.
See this issue: https://github.com/AppFlowy-IO/AppFlowy/issues/3121
In this issue, I have added a task to add a filter for grouping shortcuts according to their category.